### PR TITLE
version 0.1.2, replaces 0.1.1, set up Makefile for iterative catalog version

### PR DESCRIPTION
**Pull request by jeremyary**

### CODE REVIEW

File: config/manifests/bases/dbaas-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -5,7 +5,8 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Database
-    containerImage: quay.io/ecosystem-appeng/dbaas-operator:latest
+    com.redhat.openshift.versions: v4.8
```
Comment (tchughesiv) [inline config/manifests/bases/dbaas-operator.clusterserviceversion.yaml:8]: I don't believe we need this here... it's already been added to the `metadata/annotations.yaml`

**Review by redhatHameed — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -36,6 +46,12 @@ IMAGE_TAG_BASE ?= quay.io/ecosystem-appeng/dbaas-operator
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
+# OLD_BUNDLE_IMGS defines the comma separated list of old bundles to add to the index.
+COMMA := ,
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+OLD_BUNDLE_IMGS = $(patsubst %$(COMMA),%$(EMPTY),$(subst $(SPACE),$(EMPTY),$(foreach ver,$(subst $(COMMA),$(SPACE),$(OLD_BUNDLE_VERSIONS)),$(IMAGE_TAG_BASE)-bundle:v$(ver),)))
```
Comment (xieshenzh) [inline Makefile:53]: Please also pick the changes from this commit, it is for the change request by @redhatHameed : https://github.com/xieshenzh/dbaas-operator/commit/72d767540ed4436cd1da436250c590406a09d441

**Review by xieshenzh — CHANGES_REQUESTED** [review]

File: config/manifests/bases/dbaas-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jeremyary) [reply config/manifests/bases/dbaas-operator.clusterserviceversion.yaml]: code updated for both comments. Thanks!

File: Makefile (hunk 2 above)
Comment (jeremyary) [reply Makefile]: code updated. Take another look, please? Thanks!

**Review by tchughesiv — APPROVED** [review]
Lgtm

**Pull request merged**